app/assets/stylesheets/sufia/_file-listing.scss

```diff
@@ -5,7 +5,10 @@ h4 .small {
   color: $navbar-default-toggle-icon-bar-bg;
 }
 
-@media screen and (max-width: 767px) {
+@media screen and (max-width: 961px) {
+  #documents.table-responsive {
```

Comment:
Not a blocker, but I was wondering if this declaration works without the specificity of using the ID #documents? Using ID selectors in CSS makes them hard to override. More of a style comment.